bundle/Cache/PersistenceCachePurger.php

```diff
@@ -65,21 +75,35 @@ class PersistenceCachePurger implements CacheClearerInterface
     public function __construct(
         TagAwareAdapterInterface $cache,
         LocationHandlerInterface $locationHandler,
-        CacheIdentifierGeneratorInterface $cacheIdentifierGenerator
+        CacheIdentifierGeneratorInterface $cacheIdentifierGenerator,
+        bool $clearAllSPICacheOnSymfonyClearCache = true,
+        bool $clearAllSpiCacheFromLegacy = true
     ) {
         $this->cache = $cache;
         $this->locationHandler = $locationHandler;
         $this->cacheIdentifierGenerator = $cacheIdentifierGenerator;
+        $this->clearAllSPICacheOnSymfonyClearCache = $clearAllSPICacheOnSymfonyClearCache;
+        $this->clearAllSpiCacheFromLegacy = $clearAllSpiCacheFromLegacy;
```

Comment:
Property / variable naming is inconsistent.

Reply:
Fixed